ControlRoomApplication/ControlRoomApplication/GUI/AzimuthInputDialog.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using ControlRoomApplication.Constants;
+using ControlRoomApplication.Controllers;
+
+namespace ControlRoomApplication.GUI
+{
+    public partial class AzimuthInputDialog : InputDialog
+    {
+        private double elevationHighLimit;
+        private double elevationLowLimit;
+        private double elevationPos;
+        private double azimuthPos;
+        private Timer timer;    // Timer is responsible for constantly updating the form. 
+        private string[] values;
+        private RadioTelescopeController rtController;
+
+        public AzimuthInputDialog(RadioTelescopeController rtController)
+        {
+            timer = new Timer();
+            timer.Interval = 5;
+            timer.Tick += new EventHandler(timer_Tick);
+            timer.Start();
+
+            this.rtController = rtController;
+
+            InitializeComponent();
```

Comment:
Since we are inheriting the InputDialog form, we should probably replace `InitalizeComponent()` by calling its base constructor. That way, if we change in the future how all InputDialog forms are initialized, this one will be updated too.

- [ ] Consider calling base constructor